src/main/java/thePackmaster/cardmodifiers/gemspack/DiceGemMod.java

```diff
@@ -0,0 +1,33 @@
+package thePackmaster.cardmodifiers.gemspack;
+
+import basemod.abstracts.AbstractCardModifier;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.actions.boardgamepack.DelayedDiceRollAction;
+import thePackmaster.cardmodifiers.madsciencepack.AbstractMadScienceModifier;
+import thePackmaster.util.Wiz;
+
+public class DiceGemMod extends AbstractMadScienceModifier {
+
+    public DiceGemMod() {
+        super();
+    }
+
+    @Override
+    public String modifyDescription(String rawDescription, AbstractCard card) {
+        return rawDescription + CardCrawlGame.languagePack.getUIString(SpireAnniversary5Mod.makeID("GemModifiers")).TEXT[1];
+    }
+    @Override
+    public void onUse(AbstractCard card, AbstractCreature target, UseCardAction action) {
+        super.onUse(card, target, action);
```

Comment:
I will not highlight every usage, but I will ask you to change this everywhere in this pack. Only call the super method if it is relevant.